app/models/post_image.rb

```diff
@@ -5,6 +5,12 @@ class PostImage < ActiveRecord::Base
 
   mount_uploader :image, PostImageUploader
 
+  def image_url
```

Comment:
I thought of that, but with a null object, the call would still look like `hero.image.url.present?` which looks like a violation of the Law of Demeter, dont' you think?